### PR TITLE
Ensure the year is displayed in the block editor event single page for events that are occurring in a different year from the current one. [TEC-3740]

### PR DESCRIPTION
**Pull request by EugeneKyale**
This PR fixes the issue described in this ticket : https://theeventscalendar.atlassian.net/secure/RapidBoard.jspa?rapidView=163&projectKey=BTPLAN&modal=detail&selectedIssue=TEC-3740&assignee=5feb71d1332c3a0107acdab7

Here's a video describing what the PR entails: https://www.loom.com/share/3dbed484df6240b68558af6f452b2196

### CODE REVIEW

File: src/views/blocks/event-datetime.php [hunk 1]
```diff
@@ -35,10 +35,14 @@
 );
 
 $time_format    = tribe_get_time_format();
-$date_format    = tribe_get_date_format( $use_yearless_format );
+$date_format    = tribe_get_date_format( true );
```
Comment (paulmskim) [inline src/views/blocks/event-datetime.php:38]: It looks like a logic fault, couldn't we just do the following?

```suggestion
$date_format    = tribe_get_date_format( ! $use_yearless_format );
```

**Review by paulmskim — CHANGES_REQUESTED** [review]

**Comment by paulmskim** [comment]
It would also be good to show the case where the year shouldn't display, and make sure that's working properly

**Comment by EugeneKyale** [comment]
> It would also be good to show the case where the year shouldn't display, and make sure that's working properly

@paulmskim just checked this and found out that the events currently don't respect any date format other than F j, Y. This is the case for the classic editor as well. This probably requires a new ticket because its happening in all editors. What do you think?

File: src/views/blocks/event-datetime.php (hunk 1 above)
Comment (EugeneKyale) [reply src/views/blocks/event-datetime.php]: Yeah that neater, I've updated this.

**Review by paulmskim — DISMISSED** [review]

**Review by paulmskim — APPROVED** [review]

**Pull request merged**